### ISSUE TITLE
Tab order should be locical for next editable field

### ISSUE THREAD

**Issue by solfeggietto** [IKAMR/KDRS_SIARD_Extdoc#7]
The tab order of fields in the dialog window need be tabbing from ONE editable field to the NEXT one, in order from to fields downwards.

Typical one edit the SIARD top level metadata and want edit the next by Tab key.
But also is valid for the top folder fields.

Furtermore the buttons should Tab order similar after this.
